pkg/operator/ceph/config/livenessprobe.go

```diff
@@ -62,3 +51,27 @@ func ConfigureLivenessProbe(daemon rookv1.KeyType, container v1.Container, healt
 
 	return container
 }
+
+func GetLivenessProbeWithDefaults(desiredProbe, defaultProbe *v1.Probe) *v1.Probe {
+	// No need to allow overriding of the liveness probe command
+	desiredProbe.Handler = defaultProbe.Handler
```

Comment:
One question here with`desiredProbe.Handler = defaultProbe.Handler` if someone specifies the handler then also we will use `defaultProbe`(the default that we have in pod spec)?

Reply:
Correct, I don't see a scenario for someone to specify their own liveness probe. They can just disable it if it's not working. And if there's a bug, we should fix it.

Reply:
then it's okay have this. Thanks

Reply:
I'm a bit hesitant about this one, I understand the rationale of not allowing it (protecting users from shooting themselves in the foot and having an opinionated installation, etc) but at the same time I feel like it could become handy to allow it (something that devs would only do?). I'm not expecting the user to change the Handler anyway but focus on other values.
Each time we have tried to enforce/disallow something it somehow "backfired" a bit, we all know this feeling: "I wish I shouldn't have disallowed that option", right?

More thoughts on this?

Reply:
I'm ok if we allow overriding the liveness probe, it just didn't seem necessary. It's a fair concern that we might wish we had enabled it if we don't. :)